integration-cli/docker_cli_events_test.go

```diff
@@ -13,6 +13,44 @@ import (
 	"github.com/go-check/check"
 )
 
+func (s *DockerSuite) TestEventsSinceUntilTimestampFormats(c *check.C) {
+	image := "busybox"
+	dockerCmd(c, "run", "--rm", image, "true")
+
+	// Start stopwatch, generate some event (sleeps are due to second
+	// precision of Remote API in since/until parameters)
+	start := daemonTime(c)
+	time.Sleep(time.Second)
+	dockerCmd(c, "tag", image, "eventstest:tag1")
+	dockerCmd(c, "rmi", "eventstest:tag1")
+	time.Sleep(time.Second)
+	end := daemonTime(c)
+
+	// Generate some more events after stopping stopwatch
+	time.Sleep(time.Second)
+	dockerCmd(c, "tag", image, "eventstest:tag2")
+	dockerCmd(c, "rmi", "eventstest:tag2")
+
+	// List available time formats to --since/--until
+	unixTs := func(t time.Time) string { return fmt.Sprintf("%v", t.Unix()) }
+	rfc3339 := func(t time.Time) string { return t.Format(time.RFC3339) }
+
+	// --since=$start --until=$end must not contain 'tag2'
+	for _, f := range []func(time.Time) string{unixTs, rfc3339} {
+		since := f(start)
+		until := f(end)
+		c.Logf("since=%s until=%s", since, until)
+		cmd := exec.Command(dockerBinary, "events", "--since="+since, "--until="+until)
+		out, _, err := runCommandWithOutput(cmd)
+		if err != nil {
+			c.Fatalf("docker events cmd failed: %v\nout=%s", err, out)
+		}
+		if strings.Contains(out, "tag2") {
```

Comment:
@duglin I'm not sure what you mean:

    cmd := exec.Command(dockerBinary, "events", "--since="+since, "--until="+until)